Symfony/CS/Fixer.php

```diff
@@ -158,9 +167,51 @@ public function fix(ConfigInterface $config, $dryRun = false, $diff = false)
         return $changed;
     }
 
-    public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileCacheManager $fileCacheManager)
+    private function getNextFile(\Iterator $iterator, FileCacheManager $fileCacheManager)
```

Comment:
well, my point is precisely related to ``starting with current``. the following code will give you the same file all the time:

```php
$file = $this->getNextFile($iterator, $fileCacheManager);
$file = $this->getNextFile($iterator, $fileCacheManager);
$file = $this->getNextFile($iterator, $fileCacheManager);
$file = $this->getNextFile($iterator, $fileCacheManager);
```

But the method name ``getNextFile`` tells me that I will get the next file each time (so 4 different files)